### PR TITLE
Tests for Api.Persistent.

### PR DESCRIPTION
**Pull request by fisx**

### CODE REVIEW

File: tests/Api/PersistentSpec.hs [hunk 1]
```diff
@@ -1,17 +1,115 @@
...
+-- | the empty database
+mkEmpty :: IO (Persist :~> IO)
+mkEmpty = mkRunPersist
 
+-- | a database state containing one arbitrary item of each type (idea, user, ...)
+mkInitial :: IO (Persist :~> IO)
+mkInitial = do
+    rp <- mkEmpty
+    generate arbitrary >>= unNat rp . addIdea
+    generate arbitrary >>= unNat rp . addUser
```
Comment (np) [inline tests/Api/PersistentSpec.hs]: At some point one might need to make the users first to have the ideas tied to actual users.

**Comment by np** [comment]
LGTM

**Comment by Mikolaj** [comment]
LGTM (apart of the travis complaints)

**Comment by andorp** [comment]
I would wait for the fix.

**Pull request merged**